web-console/src/lib/services/manager/core/request.ts

```diff
@@ -108,7 +110,7 @@ export const getFormData = (options: ApiRequestOptions): FormData | undefined =>
       if (isString(value) || isBlob(value)) {
         formData.append(key, value)
       } else {
-        formData.append(key, JSON.stringify(value))
+        formData.append(key, JSONbig.stringify(value))
```

Comment:
why do we use JSONbig now?

Reply:
The spec for max_output_buffer_time_millis and max_output_buffer_time_records fields dictates the type usize (u64), and default JSON serialization doesn't support 64-bit integers (it truncates them to a floating point number).
So here I am changing the auto-generated code for request serialization (a supported feature of openapi-typescript codegen) to parse wire JSON with a library that spits out BigNumber-s if it doesn't fit in a built-in JavaScript Number.

64-bit integers will probably get used elsewhere in API, so this change will be more useful as the time goes on.